projects/storefrontlib/layout/main/storefront.component.html

```diff
@@ -3,12 +3,22 @@
     <header
       id="cx-header"
       cxSkipLink="cx-header"
-      [cxFocus]="{ disableMouseFocus: true }"
+      [cxFocus]="focusConfig"
       [class.is-expanded]="isExpanded$ | async"
       (keydown.escape)="collapseMenu()"
       (click)="collapseMenuIfClickOutside($event)"
     >
-      <cx-page-layout section="header"></cx-page-layout>
+      <ng-container *cxFeature="'a11yHamburgerMenuTrapFocus'">
+        <cx-page-layout
+          [cxSkipFocus]="skipFocusConfig"
```

Comment:
Because the `skipFocusConfig` will be changing frequently, why don't we use `skipFocusConfig$ | async` instead? Then we could use an `Observable` object instead in our class instead of making another subscription.

Reply:
On this note and about responsiveness to changes as above and readability, maybe the `isEnabled` should be its own parameter. There could be arguments against this though. What do you think?